cmd/docgen/templates/docs.md

```diff
@@ -171,11 +171,10 @@ The flow context contains the following variables:
  * `run.parent.results.[snaked_result_name]` the value of a specific value of the run that started this run
  * `run.parent.results.[snaked_result_name].category` the category (if any) of a specific value of the run that started this run
 
- * `input` values of the last input from the contact, shorthand for input.text
- * `input.text` the text of the last input from the contact
- * `input.urn` the urn that was used to send the last input
- * `input.contact_uuid` the uuid of the contact that sent the last input
- * `input.channel_uuid` the uuid of the channel that was used to send the last input
+ * `run.input` values of the last input from the contact
+ * `run.input.text` the text of the last input from the contact
+ * `run.input.urn` the urn that was used to send the last input
+ * `run.input.channel` the channel that was used to send the last input
```

Comment:
I don't see `channel` in the our `Resolve` for MsgInput, so either this or that implementation seem wrong.